### PR TITLE
Add CommerceJS Hacktoberfest swag

### LINKED ISSUE #764: CommerceJS Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : CommerceJS 

Q. Where have you seen the announcement (URL)?  
A. : https://commercejs.com/docs/hacktoberfest/

Q. What kind of swag (clothing, stickers, ...)?  
A. : undisclosed

- [ ] I've checked that the promotion is legit.
- [ ] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by GuyKh**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Add CommerceJS Hacktoberfest swag

Fixes #764

### CODE REVIEW

**Comment by GuyKh** [comment]
Tagged `unknown` instead of `undisclosed` -- 
I've already committed a PR and will update upon me understanding what they're sending

File: data.json [hunk 1]
```diff
@@ -709,5 +709,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
```
Comment (aslafy-z) [inline data.json:712]: Please move this item alphabetically according to the name.

**Comment by aslafy-z** [comment]
Prefer using `Closes` or `Fixes` keyword to link to the issue, see https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: data.json (hunk 1 above)
Comment (GuyKh) [reply data.json]: 👍🏻

**Comment by GuyKh** [comment]
Unittest failed on `UNO`

**Comment by aslafy-z** [comment]
One more run fixed the issue. We should ensure that these http calls are retried if they fail. Thanks!

**Review by aslafy-z — APPROVED** [review]
LGTM

**Review by vikaspotluri123 — APPROVED** [review]

**Pull request merged**